src/services/chat/prepopulated.js

```diff
@@ -48,7 +48,8 @@ module.exports = async (id, users) => {
             other_text: textTargetUser,
             own_text: textOwnUser,
             system_user: id,
-            isSystem: true
+            isSystem: true,
+            type: 'system'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `type: 'system'` property seems redundant as there is already an `isSystem: true` property. If the `type` property is meant to provide more granular control over the type of system message, consider using more descriptive values than `'system'`. However, if it's just meant to indicate that this is a system message, it might be unnecessary given the presence of `isSystem`.
```diff
             other_text: textTargetUser,
             own_text: textOwnUser,
             system_user: id,
             isSystem: true,
-            type: 'system'
           },
           {
             skip_push: true
```
If you decide to keep both properties, please ensure they are used consistently throughout the codebase to avoid confusion and potential bugs.


<!-- This is an auto-generated comment by OSS CodeRabbit -->